lib/android-helpers.js

```diff
@@ -890,5 +891,57 @@ helpers.isEmulator = function isEmulator (adb, opts) {
 helpers.bootstrap = Bootstrap;
 helpers.unlocker = unlocker;
 
-export { helpers, SETTINGS_HELPER_PKG_ID, prepareAvdArgs, ensureNetworkSpeed };
+const CONSTRAINTS = {
+  reboot: {
+    isBoolean: true,
+    presence: true
```

Comment:
is it really mandatory that these values are present?

Reply:
I'd assume so because usually appium passes in default arg values to the driver in the case none were given. Now in appium 2.0 we are trying to depreciate driver specific args. So once the parsing is done I'd assume we want validateCaps to ensure that the args are present(they could be either default or user passed in values)